src/components/Forms/Upload.jsx

```diff
@@ -1,14 +1,34 @@
 import React, { useState } from "react";
 import { BsUpload } from "react-icons/bs";
 import { FaFilePdf, FaTimes } from "react-icons/fa";
-
-const Upload = ({ field, user, setUser, text }) => {
+import { toast } from "react-hot-toast";
+const bytes = {
+  B: 1,
+  KB: 1024,
+  MB: 1048576,
+  GB: 1073741824,
+};
+const getSize = (maxSize) => {
+  let size = 0;
+  Object.entries(maxSize).forEach(([byte, num]) => (size += bytes[byte] * num));
```

Comment:
just return this directly, if size gets returned as 0 then we have issues in our codebase since this is internally handled